pkg/backends/calico/calico_test.go

```diff
@@ -0,0 +1,146 @@
+package calico
```

Comment:
I'd suggest renaming this file to `routes_test.go` since it is primarily to test function from `routes.go`.

Generally good to have 1:1 test/code